### PR TITLE
BTFSBTT-189: Display metadata from reading object

### PR DESCRIPTION
**Pull request by steveyeom**
This PR includes changes for BTFSBTT-189 and 242 (Remove token metadata from the output of btfs commands) and also minor changes for trickle.

### CODE REVIEW

File: importer/helpers/dagbuilder.go [hunk 1]
```diff
@@ -207,7 +208,9 @@ func (db *DagBuilderHelper) FillNodeLayer(node *FSNodeOverDag) error {
 		if err != nil {
 			return err
 		}
-
+		if db.tokenMetaToProcess {
+			db.SetTokenMetaToProcess(false)
```
Comment (taiyangc) [inline importer/helpers/dagbuilder.go:212]: why set to false?

**Review by taiyangc — COMMENTED** [review]
just one question, otherwise looks good

**Review by taiyangc — APPROVED** [review]
lgtm

File: importer/helpers/dagbuilder.go (hunk 1 above)
Comment (steveyeom) [reply importer/helpers/dagbuilder.go:212]: Talked to Eric

**Pull request merged**